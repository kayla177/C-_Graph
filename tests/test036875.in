ENTITY 3LR62DDT name_3LR6 journal
ENTITY KQ5MLXBQ name_KQ5M author
ENTITY 4G6U3CSU name_4G6U author
ENTITY S82LR233 name_S82L journal
ENTITY XBDLFEB5 name_XBDL institution
ENTITY WRZ8LORG name_WRZ8 author
ENTITY J9DDJ34R name_J9DD journal
ENTITY HGVD0HKD name_HGVD paper
ENTITY CY1AVORF name_CY1A conference
ENTITY HJ408U0B name_HJ40 author
RELATIONSHIP CY1AVORF presented_at S82LR233 9.8
RELATIONSHIP S82LR233 presented_at 3LR62DDT 2.1
RELATIONSHIP XBDLFEB5 collaborated_with S82LR233 0.4
RELATIONSHIP 4G6U3CSU cited_by KQ5MLXBQ 9.8
RELATIONSHIP WRZ8LORG collaborated_with J9DDJ34R 0.4
RELATIONSHIP XBDLFEB5 published_in J9DDJ34R 1.8
RELATIONSHIP J9DDJ34R collaborated_with KQ5MLXBQ 3.6
RELATIONSHIP KQ5MLXBQ affiliated_with S82LR233 2.8
RELATIONSHIP KQ5MLXBQ collaborated_with HJ408U0B 5.9
RELATIONSHIP J9DDJ34R presented_at 4G6U3CSU 5.1
RELATIONSHIP 4G6U3CSU affiliated_with 3LR62DDT 7.7
RELATIONSHIP WRZ8LORG affiliated_with HJ408U0B 1.9
RELATIONSHIP KQ5MLXBQ authored_by 3LR62DDT 3.4
RELATIONSHIP XBDLFEB5 authored_by HGVD0HKD 5.0
RELATIONSHIP XBDLFEB5 authored_by KQ5MLXBQ 5.4
PATH KQ5MLXBQ 3LR62DDT
HIGHEST
HIGHEST
DELETE XBDLFEB5
HIGHEST
PATH J9DDJ34R 3LR62DDT
PRINT CY1AVORF
PRINT CY1AVORF
HIGHEST
PRINT WRZ8LORG
EXIT
