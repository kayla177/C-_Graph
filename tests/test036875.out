success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
KQ5MLXBQ 4G6U3CSU 3LR62DDT 17.5
3LR62DDT HGVD0HKD 27.9
3LR62DDT HGVD0HKD 27.9
success
3LR62DDT WRZ8LORG 25.3
J9DDJ34R 4G6U3CSU 3LR62DDT 12.8
S82LR233
S82LR233
3LR62DDT WRZ8LORG 25.3
J9DDJ34R HJ408U0B
